manifests/profile/prometheus/exporter/mysql.pp

```diff
@@ -25,5 +25,12 @@
     content => template('nebula/profile/prometheus/exporter/mysql/defaults.sh.erb')
   }
 
+  @@concat_fragment { "prometheus mysql service ${::hostname}":
+    tag     => "${::datacenter}_prometheus_mysql_service_list",
+    target  => '/etc/prometheus/mysql.yaml',
+    content => template('nebula/profile/prometheus/exporter/mysql/target.yaml.erb')
+  }
+
+  Firewall <<| tag == "${::datacenter}_prometheus_mysql_exporter" |>>
```

Comment:
This will import all `firewall` resources with the tag `${::datacenter}_prometheus_mysql_exporter`, and currently no such resource is being exported anywhere, so it will find 0 results. (The prometheus profile needs to export it.)

Reply:
I think i understand what was missing here.  Will wait till my next PR passes, to validate that with you.

Reply:
Yeah you got it.